schemas/com.sendgrid/bounce/jsonschema/2-0-1

```diff
@@ -0,0 +1,105 @@
+  "description": "Schema for a SendGrid bounce event. Property descriptions derived from the SendGrid documentation: https://sendgrid.com/docs/for-developers/tracking-events/event/",
+  "self": {
+    "vendor": "com.sendgrid",
+    "name": "bounce",
+    "version": "2-0-1",
+    "format": "jsonschema"
+  },
+  "type": "object",
+  "properties": {
+    "ip": {
+      "description": "The IP address used to send the email. For open and click events, it is the IP address of the recipient who engaged with the email.",
+      "type": "string"
+    },
+    "timestamp": {
+      "description": "The timestamp of when the message was sent",
+      "type": "string",
+      "format": "date-time"
+    },
+    "email": {
+      "description": "The email address of the recipient",
+      "type": "string"
+    },
+    "newsletter": {
+      "description": "Legacy Marketing Email tool fields",
+      "type": "object",
+      "properties": {
+        "newsletter_user_list_id": {
+          "type": "string"
+        },
+        "newsletter_id": {
+          "type": "string"
+        },
+        "newsletter_send_id": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": true
+    },
+    "sg_event_id": {
+      "description": "A unique ID to this event that you can use for deduplication purposes. These IDs are up to 100 characters long and are URL safe.",
+      "type": "string",
+      "minLength": 22,
+      "maxLength": 4096
+    },
+    "smtp-id": {
+      "description": "A unique ID attached to the message by the originating system",
+      "type": "string"
+    },
+    "reason": {
+      "description": "Any sort of error response returned by the receiving server that describes the reason this event type was triggered",
+      "type": "string"
+    },
+    "tls": {
+      "description": "Indicates whether TLS encription was used in sending this message. For more information about TLS, see the SendGrid TLS Glossary page.",
+      "type": ["string", "integer"]
+    },
+    "status": {
+      "description": "Status code string. Corresponds to HTTP status code - for example, a JSON response of 5.0.0 is the same as a 500 error response.",
+      "type": "string"
+    },
+    "cert_error": {
+      "description": "No longer found in the SendGrid documentation; possibly deprecated",
+      "type": "string"
+    },
+    "category": {
+      "description": "Categories are custom tags that you set for the purpose of organizing your emails. Categories can be set as an array or string, and they will be returned as such when posted in your event endpoint.",
+      "type": ["array", "string"],
+      "items": {
+        "type": "string"
+      }
+    },
+    "asm_group_id": {
+      "description": "The ID of the unsubscribe group the recipient’s email address is included in. ASM IDs correspond to the ID that is returned when you create an unsubscribe group.",
+      "type": "integer",
```

Comment:
I think removing or changing the maximum might make this a breaking change that requires us to make it a `3-0-0` version instead of `2-0-1`.

Also, we could consider keeping maximum, just making it higher – perhaps 2147483647 make it a 32-bit integer or even 9223372036854775807 if we think we need it.